README.adoc

```diff
@@ -141,25 +147,30 @@ The `@Timed` annotation applies to the `get()` method to track how frequently th
 invoked and also how long it takes for the invocation to complete. This annotation is
 followed by fields that correspond to metadata fields:
 
-* The `name` field is optional and sets the name of the metric.
-* The `unit` field has a default value of `seconds`.
-* The `description` field is optional and describes the purpose of the metric
-annotation.
+|===
+| *Field* | *Purpose*
+|`name` | Optional and sets the name of the metric.
+|`description` | Optional and describes the purpose of the metric annotation.
+|===
```

Comment:
Missing `unit` field?

I am not sure if we need a header row.

Reply:
The `unit` field was added somehow to @Timed annotation even though we did not use it. so I removed it.